sql/moz-fx-data-shared-prod/telemetry_derived/fx_health_ind_clients_daily_by_os_v1/query.sql

```diff
@@ -0,0 +1,94 @@
+WITH searches_per_user_by_os_and_date_staging AS (
+  SELECT
+    submission_date_s3,
+    os,
+    SUM(search_count_all) AS searches,
+    COUNT(DISTINCT client_id) AS users,
+  FROM
+    `moz-fx-data-shared-prod.telemetry.clients_daily`
+  WHERE
+    submission_date_s3 = @submission_date
+    AND app_name = 'Firefox'
+    AND sample_id = 42
+    AND search_count_all < 10000
```

Comment:
same as above. Do we need the `search_count_all` filtered

Reply:
I think it was to avoid skewing results by removing all the really excessive ones